services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/model/WriteBatchTest.java

```diff
@@ -134,6 +138,16 @@ public void builder_missing_mapped_table_resource_error_message() {
                                                      errorMessageDeleteKeyFromItem);
     }
 
+    @Test
+    public void hashCode_includesConsumedCapacity() {
```

Comment:
Can we use equalsVerifier to test equals and hashcode? https://github.com/aws/aws-sdk-java-v2/blob/master/core/sdk-core/src/test/java/software/amazon/awssdk/core/async/ResponsePublisherTest.java#L25

We may need to add `equalsverifier` test dependency.